### PR TITLE
Update installation documentation page

### PR DESCRIPTION
**Pull request by Meakk**

### CODE REVIEW

File: doc/user/INSTALLATION.md [hunk 1]
```diff
@@ -1,20 +1,43 @@
...
 - [Arch User Repository](https://aur.archlinux.org/packages/f3d)
-- [FreeBSD](https://cgit.freebsd.org/ports/tree/graphics/f3d)
 - [NixOS](https://search.nixos.org/packages?channel=22.05&show=f3d&from=0&size=50&sort=relevance&type=packages&query=f3d)
 - [openSUSE](https://software.opensuse.org/package/f3d)
 - Fedora and others through [openSUSE OBS](https://build.opensuse.org/package/show/home:AndnoVember:F3D/f3d)
-- [MacOS Homebrew](https://formulae.brew.sh/formula/f3d)
-- [Windows Scoop](https://scoop.sh/#/apps?q=f3d&s=0&d=1&o=true)
+- [Guix](https://guix.gnu.org/en/packages/f3d-1.3.1/)
+
+### Other
```
Comment (Meakk) [inline doc/user/INSTALLATION.md:39]: Not sure if Flatpack and Spack should be under Other or Linux, what do you think @mwestphal ?

File: doc/user/INSTALLATION.md [hunk 2]
```diff
@@ -1,20 +1,43 @@
...
 
 You can also find packages for the following package managers:
 
+### Windows
+
+- [Scoop](https://scoop.sh/#/apps?q=f3d&s=0&d=1&o=true)
+
+### MacOS
+
+- [Homebrew](https://formulae.brew.sh/formula/f3d)
```
Comment (mwestphal) [inline doc/user/INSTALLATION.md:27]: Brew is also available on linux though.

File: doc/user/INSTALLATION.md (hunk 1 above)
Comment (mwestphal) [reply doc/user/INSTALLATION.md:39]: Flatpack is linux specific, spack is not (supports linux, macos and strange HPC stuff).

File: doc/user/INSTALLATION.md [hunk 3]
```diff
@@ -1,20 +1,43 @@
...
 
-You can find the binary release packages for Windows, Linux and macOS on the [releases page](https://github.com/f3d-app/f3d/releases).
+## Current release packages (1.3.1)
+
+| Platform | Files |
+| -------- | ----- |
+| Windows Installer | [f3d-1.3.1-win64.exe](https://github.com/f3d-app/f3d/releases/download/v1.3.1/f3d-1.3.1-win64.exe) |
+| Windows (Portable) | [f3d-1.3.1-win64.zip](https://github.com/f3d-app/f3d/releases/download/v1.3.1/f3d-1.3.1-win64.zip) |
+| MacOS Installer | [f3d-1.3.1-Darwin.dmg](https://github.com/f3d-app/f3d/releases/download/v1.3.1/f3d-1.3.1-Darwin.dmg) |
+| Debian-based Linux Installer | [f3d-1.3.1-Linux.deb](https://github.com/f3d-app/f3d/releases/download/v1.3.1/f3d-1.3.1-Linux.deb) |
```
Comment (mwestphal) [inline doc/user/INSTALLATION.md:10]: Debian package ?

File: doc/user/INSTALLATION.md [hunk 4]
```diff
@@ -1,20 +1,43 @@
...
+| Linux (Portable) | [f3d-1.3.1-Linux.tar.xz](https://github.com/f3d-app/f3d/releases/download/v1.3.1/f3d-1.3.1-Linux.tar.xz) |
```
Comment (mwestphal) [inline doc/user/INSTALLATION.md:11]: Did you to inject some javascripts ?

File: doc/user/INSTALLATION.md [hunk 5]
```diff
@@ -1,20 +1,43 @@
...
+
+## Alternatives
+
+You can find the older binary release packages for Windows, Linux and macOS on the [releases page](https://github.com/f3d-app/f3d/releases).
 See the [desktop integration](DESKTOP_INTEGRATION.md) section in order actually integrate the binary release in your desktop.
 Alternatively, you can build F3D yourself by following the [build](dev/BUILD.md) guide.
 
 You can also find packages for the following package managers:
```
Comment (mwestphal) [inline doc/user/INSTALLATION.md:19]: technically not only package manager. repology calls them "repository".

**Comment by mwestphal** [comment]
There is a link to releases in the README.md that should be updated.

File: doc/user/INSTALLATION.md (hunk 2 above)
Comment (Meakk) [reply doc/user/INSTALLATION.md:27]: Yes but nobody is using brew on linux

File: doc/user/INSTALLATION.md (hunk 3 above)
Comment (Meakk) [reply doc/user/INSTALLATION.md:10]: It works on Ubuntu too

File: doc/user/INSTALLATION.md (hunk 4 above)
Comment (Meakk) [reply doc/user/INSTALLATION.md:11]: No, it is done manually for now

File: doc/user/INSTALLATION.md (hunk 3 above)
Comment (mwestphal) [reply doc/user/INSTALLATION.md:10]: Debian-based package ?

File: doc/user/INSTALLATION.md (hunk 4 above)
Comment (mwestphal) [reply doc/user/INSTALLATION.md:11]: ok

File: doc/user/INSTALLATION.md (hunk 2 above)
Comment (mwestphal) [reply doc/user/INSTALLATION.md:27]: as you see fit

File: doc/user/INSTALLATION.md (hunk 3 above)
Comment (mwestphal) [reply doc/user/INSTALLATION.md:10]: What i mean is that it is not an installer.

File: doc/user/INSTALLATION.md [hunk 6]
```diff
@@ -1,20 +1,43 @@
 # Installation
 
-You can find the binary release packages for Windows, Linux and macOS on the [releases page](https://github.com/f3d-app/f3d/releases).
+## Current release packages (1.3.1)
+
+| Platform | Files |
+| -------- | ----- |
+| Windows Installer | [f3d-1.3.1-win64.exe](https://github.com/f3d-app/f3d/releases/download/v1.3.1/f3d-1.3.1-win64.exe) |
+| Windows (Portable) | [f3d-1.3.1-win64.zip](https://github.com/f3d-app/f3d/releases/download/v1.3.1/f3d-1.3.1-win64.zip) |
+| MacOS Installer | [f3d-1.3.1-Darwin.dmg](https://github.com/f3d-app/f3d/releases/download/v1.3.1/f3d-1.3.1-Darwin.dmg) |
```
Comment (mwestphal) [inline doc/user/INSTALLATION.md:9]: not sure a dmg is an installer, how do MacOS users calls them ?

**Comment by mwestphal** [comment]
Should we move this one forward as is ?

**Comment by Meakk** [comment]
I forgot about this one. I'll take a look later

File: doc/user/INSTALLATION.md (hunk 6 above)
Comment (Meakk) [reply doc/user/INSTALLATION.md:9]: I kept only "MacOS"

**Review by mwestphal — APPROVED** [review]

**Pull request merged**